### PR TITLE
chore: default the writer of docker build to `Stderr` incase of single image

### PR DESCRIPTION
**Pull request by KollaAdithya**
related #5124, #5067
This PR will remove the rolling display when there is only one image to build.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/deploy/workload.go [hunk 1]
```diff
@@ -401,6 +401,7 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 	if len(buildArgsPerContainer) == 0 {
 		return nil
 	}
+	isMultipleContainerImages := len(buildArgsPerContainer) != 1
```
Comment (huanjani) [inline internal/pkg/cli/deploy/workload.go]: maybe??
```suggestion
	isMultipleContainerImages := len(buildArgsPerContainer) > 1
```

**Comment by huanjani** [comment]
Could we add a unit test for multiple images?

**Comment by KollaAdithya** [comment]
I think we already had a test case for this scenario of multiple images over here
https://github.com/aws/copilot-cli/pull/5148/files#diff-cc7a53c33c7639692deae85af8e5ec1187f3ef5dc845b29f53f9ae698043ef7cR311

**Review by huanjani — APPROVED** [review]
lgtm!

File: internal/pkg/cli/deploy/workload.go [hunk 2]
```diff
@@ -401,6 +401,7 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 	if len(buildArgsPerContainer) == 0 {
 		return nil
 	}
+	isMultipleContainerImages := len(buildArgsPerContainer) > 1
```
Comment (iamhopaul123) [inline internal/pkg/cli/deploy/workload.go]: Does it make more sense for us to call a separate function if `isMultipleContainerImages` is true (after `repository.Login()`)? The reason is we probably don't need the parallel part at all for single build case and it'll be easier to read/maintain

**Review by iamhopaul123 — APPROVED** [review]
awesome :shipit:

**Pull request merged**